### ISSUE TITLE
FR: please co-publish on Open VSX Registry

### ISSUE THREAD

**Issue by GitMensch** [hashhashu/sqltools-Oracle-driver#12]
Please add this useful extension to the [Open VSX Registry](https://open-vsx.org/?search=oracle).

Details on the publishing can be found at https://github.com/eclipse/openvsx/wiki/Publishing-Extensions.

A bit of explanation:
Open VSX Registry is the default extension provider endpoint for the free build of vscode VSCodium and Eclipse Theia, as well as browser versions like the one on GitPod. It would be nice to have this extension be directly published to a free vsix repository instead of only the Microsoft Marketplace with its restrictions.

You could also use a [GitHub action](https://docs.github.com/en/actions) that allows publishing to Open VSX and the MS market place, [HaaLeo/publish-vscode-extension](https://github.com/HaaLeo/publish-vscode-extension#readme) does this.

**Comment by Yahud100** [comment]
yes i am also looking for this , it will be great if added into open vsx registry.

hope it will be added soon.

**Comment by hashhashu** [comment]
I just publish it in https://open-vsx.org/extension/hurly/sqltools-oracle-driver, but i doesn't tested it yet.
Fee free to ask if you have any questions, thanks
